packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -103,6 +103,12 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
    * Call to provide customized styling that will layer on top of the variant rules.
    */
   getStyles?: IStyleFunction<ISearchBoxStyleProps, ISearchBoxStyles>;
+
+  /**
+   * Whether or not to animate the SearchBox icon on focus.
+   * @default true
+   */
+  animateIcon?: boolean;
```

Comment:
this should be 'disableAnimation' so that the default value is false and invoking it is done via <SearchBox disableAnimation />